task/fossa-scanner/0.1/tests/run.yaml

```diff
@@ -0,0 +1,38 @@
+---
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: fossa-api-key-sec
+type: Opaque
+data:
+  apikey: "ZTgwNzg3MmE0NGVkNDFlNGY5OTdlNmVlZmQyNzI2ZjE="
+
+---
+
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: fossa-scan-springboot
+spec:
+  taskRef:
+    name: fossa-scanner
+  resources:
+    inputs:
+      - name: repo
+        resourceSpec:
+          type: git
+          params:
+            - name: revision
+              value: main
+            - name: url
+              value: https://github.com/spring-projects/spring-boot.git
```

Comment:
this can probably be replaced with git-clone task and TaskRun can be converted into the PipelineRun